schema/scroll.xsd

```diff
@@ -0,0 +1,35 @@
+<xs:schema
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  attributeFormDefault="qualified"
+  elementFormDefault="qualified"
+  targetNamespace="https://hyperview.org/hyperview-scroll"
+  xmlns:hv="https://hyperview.org/hyperview"
+>
+  <xs:import
+    namespace="https://hyperview.org/hyperview"
+    schemaLocation="hyperview.xsd"
+  />
+
+  <xs:element name="option">
+    <xs:complexType>
+      <xs:sequence minOccurs="0" maxOccurs="unbounded">
+          <xs:element ref="hv:behavior" />
+      </xs:sequence>
+      <xs:attribute name="label" type="xs:string" use="required" />
+      <xs:attributeGroup ref="hv:behaviorAttributes" />
+    </xs:complexType>
+  </xs:element>
```

Comment:
I think this is leftover from copying?

Reply:
yes good catch

Reply:
[cf7106d](https://github.com/Instawork/hyperview/pull/669/commits/cf7106d742c4b1f4f2682a3f9bc53fb23e0a1639)